instrumentation/opentelemetry-instrumentation-aiohttp-client/tests/test_aiohttp_client_integration.py

```diff
@@ -188,7 +188,7 @@ def test_span_name_option(self):
                     [
                         (
                             expected,
-                            (StatusCanonicalCode.OK, None),
+                            (StatusCode.UNSET, None),
```

Comment:
Ah, it looks like the instrumentation never set it to begin with.

If that's the case I'd prefer adding the call: https://github.com/open-telemetry/opentelemetry-python/blob/master/instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py#L187, but understand that this may be the current intended behavior of the Spec / PR.

Reply:
Might have been a bit unclear in the original issue description, instrumentations and trace api/sdk are no longer responsible for setting Status.OK on spans. Users will need to do it themselves. Updated the description to reflect that.

Reply:
fair enough!